src/app/books/bestSeller/page.tsx

```diff
@@ -1,5 +1,41 @@
-const bestSeller = () => {
-  return <h1>여기에 bestSeller가 보여집니다.</h1>;
+'use client';
+import { useEffect, useState } from 'react';
+
+import { getBestsellerRequest, resetBooks } from '@/app/actions/types';
+import { RootState } from '@/app/reducers';
+import { Button, Container } from '@mui/material';
+import { useSelector, useDispatch } from 'react-redux';
+
+import BestSellerContainer from '../../components/Book/BestSellerContainer';
+import { AppDispatch } from '../../store/store';
+
+const BestSeller = () => {
+  const [page, setPage] = useState<number>(1);
+  const dispatch = useDispatch<AppDispatch>();
+  const { books } = useSelector((state: RootState) => state.book);
+
+  useEffect(() => {
+    dispatch(resetBooks());
+    dispatch(getBestsellerRequest('Bestseller', page, 20));
```

Comment:
I think you could also extract the GroupName via useRouter, otherwise it is not so dynamic here.